<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointPassportFormControl</class>
 <widget class="QWidget" name="JointPassportFormControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="depthVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>Depth MD, m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="depthSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="fieldVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="fieldLabel">
       <property name="text">
        <string>Field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fieldComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="dateVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="depthTVDVerticalLayout" stretch="0,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="depthTVDLabel">
       <property name="text">
        <string>Depth TVD, m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="depthTVDSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="wellCluesterVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="wellClusterLabel">
       <property name="text">
        <string>Well cluster</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wellClusterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="wellVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="wellLabel">
       <property name="text">
        <string>Well</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wellComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldComboBox</tabstop>
  <tabstop>wellComboBox</tabstop>
  <tabstop>wellClusterComboBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>depthSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
